api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -153,6 +153,22 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterVPCSubnetNames() (allErrs f
 	return allErrs
 }
 
+func (r *IBMPowerVSCluster) validateIBMPowerVSClusterVPCSubnets() (allErrs field.ErrorList) {
+	if err := r.validateIBMPowerVSClusterVPCSubnetNames(); err != nil {
+		allErrs = append(allErrs, err...)
+	}
+
+	vpcZones, err := regionUtil.VPCZonesForVPCRegion(*r.Spec.VPC.Region)
```

Comment:
VPC can be nil, Ref PR https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/2038, Safe to add checks

Reply:
Done